site/config.toml

```diff
@@ -49,6 +49,12 @@ disqusShortname = "checkly"
 	pre="/docs/images/icons/alerting.svg"
 	weight = 50
 
+[[menu.docs]]
+	name = "Monitoring-as-Code"
```

Comment:
Checkly Docs / Terraform Provider?
I believe that it makes sense to use the 'Terraform Provider' in the menu structure for now, mostly because it's the only way to do MaC currently. -> MaC is the marketing term